Library/Formula/getdns.rb

```diff
@@ -21,7 +22,10 @@ class Getdns < Formula
   depends_on "libev" => :optional
 
   def install
-    args = []
+    args = [
+      "--with-ssl=#{Formula["openssl"].opt_prefix}",
+      "--with-trust-anchor=/usr/local/etc/getdns-root.key"
```

Comment:
Thanks - it's good now - somehow I missed the docs @ https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md#variables-for-directory-locations